runtime/rococo/src/lib.rs

```diff
@@ -79,6 +79,7 @@ pub mod constants;
 mod tests;
 pub mod weights;
 pub mod xcm_config;
+// pub mod migration;
```

Comment:
what's this line? And the following line 127 ?

Reply:
Temporarily turn on the migration function on rococo, and then turn it off